.github/workflows/osv-scanner-unified-workflow.yml

```diff
@@ -25,6 +25,8 @@ on:
     branches: ["main"]
 
 permissions:
+  # Required to upload SARIF file to CodeQL
+  actions: read
```

Comment:
Can you link to the issue you posted as well here? We'll want to remove this once that fix is merged in.

Reply:
done